lib/beaker/options/command_line_parser.rb

```diff
@@ -195,8 +195,14 @@ def initialize
             @cmd_options[:validate] = bool
           end
 
-          opts.on '--collect-perf-data', 'Use sysstat on linux hosts to collect performance and load data' do
-            @cmd_options[:collect_perf_data] = true
+          opts.on '--collect-perf-data [MODE]',
+                  'Collect SUT performance and load data',
+                  'Possible values:',
+                  'aggressive (poll every minute)',
+                  'normal (poll every 10 minutes)',
+                  'none (do not collect perf data)',
+                  '(default: none)' do |mode|
```

Comment:
> Shouldn't this default be 'normal' ?

I wasn't sure about that, actually.  The default collection is "none" unless the flag is specified, but if the flag is specified without an argument, it will use "normal" collection.

I've changed the docs and updated this PR.